commons/passive/dreem/dreem_hypnogram.avsc

```diff
@@ -0,0 +1,15 @@
+{
+  "type": "record",
+  "namespace": "org.radarcns.passive.dreem",
+  "name": "Dreem",
+  "doc": "AVRO Schema for DREEM hypnogram data",
+  "fields": [
+    {"name": "studyId", "type":"string", "doc":"This is study number"},
+    {"name": "subjId", "type":"string", "doc":"Subject Identifier for the study. Format: xxx-yyy where xxx represents 3 digits for the site ID and yyy represents 3 digits for the participant ID"},
+	{"name": "epochNumber", "type":"int", "doc":"Interval (30 second) of  measurement per subject"},
+    {"name": "recDateTime", "type":"string","logicalType":"timestamp-millis", "doc":"Date & time corresponding to the recording start. Format: (ISO8601) YYYY-MM-DD’T’HH:MM:SS[Z/+hhmm/-hhmm]"},
```

Comment:
Please use the `time` variable to describe what time this data record is about and `timeReceived` to map when the record was processed. `recDateTime` can be removed. See the README for the full contribution guidelines. Or look at other schema’s to get an idea of how the schema looks like.